src/components/Charts/circleCorners.js

```diff
@@ -0,0 +1,147 @@
+/* THIS FILE IS FROM A NON-TYPESCRIPT LIBRARY THAT WOULDN'T IMPORT PROPERLY
```

Comment:
We cannot use this file without proper attribution (it is MIT licensed).
Also, this file is in the package.json, but not used through there.

Can we try to make the package.json inclusion work, instead of copying source over?

Reply:
zach is owning this too

Reply:
removed the copy of this file. we are now using the package from the registry. https://github.com/Uniswap/interface/pull/4288